addons/enemybar/targetBar.lua

```diff
@@ -0,0 +1,102 @@
+--[[
+Copyright © 2015, Mike McKee
+All rights reserved.
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of enemybar nor the
+      names of its contributors may be used to endorse or promote products
+      derived from this software without specific prior written permission.
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL Mike McKee BE LIABLE FOR ANY
+DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+--]]
+
+tbg_cap_l = images.new(tbg_cap_settings)
+tbg_cap_r = images.new(tbg_cap_settings)
+tbg_body = images.new(tbg_body_settings)
+tfgg_body = images.new(tfgg_body_settings)
+tfg_body = images.new(tfg_body_settings)
+t_text = texts.new('  ${name|(Name)} - HP: ${hpp|(100)}% ${debug|}', settings.textSettings, settings)
+
+init_target_images = function(...)
+    tbg_cap_l:pos(settings.pos.x - 1, settings.pos.y)
+    tbg_cap_r:pos(settings.pos.x + settings.targetBarWidth, settings.pos.y)
+end
+
+timer = 0
+
+render_target_bar = function (...)
+    if visible == true then		
+        tbg_cap_l:show()
+        tbg_cap_r:show()
+        tbg_body:show()
+        tfg_body:show()
+        tfgg_body:show()
+        t_text:show()
+        
+        local target = windower.ffxi.get_mob_by_target('t')
+        local player = windower.ffxi.get_player()
+        local party = windower.ffxi.get_party()
```

Comment:
Same as above, `windower.ffxi.get_party` does not seem to be needed at all, so you should remove it from here.